app/src/focus.ts

```diff
@@ -0,0 +1,27 @@
+export interface StringTMap<T> { [key: string]: T; };
+export interface StringAnyMap extends StringTMap<any> {};
+export interface StringBooleanMap extends StringTMap<boolean> {};
+
+export class Focus {
```

Comment:
Can you help add some comment describing the purpose of this class?

Reply:
Hum... 

`StringAnyMap` and `StringBooleanMap` are only interfaces that I often use when I need typings in objects. They should not be exported.

Reply:
@bigopon maybe you were talking about the `Focus` class ? It's only the view-model for the `focus` custom element in the test app. I only wanted to have a dedicated code source to test the focus attribute and create a simple custom element for this.